packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -46,6 +46,27 @@ const postCSSLoaderOptions = {
   ],
 };
 
+// style files regexes
+const CSSRegex = /\.css$/;
+const CSSModuleRegex = /\.module\.css$/;
+const SASSRegex = /\.(scss|sass)$/;
+const SASSModuleRegex = /\.module\.(scss|sass)$/;
+
+// Common function to create any style loader
+const getStyleLoader = (CSSLoaderOptions, firstLoader, firstLoaderOptions) => {
```

Comment:
I'm not sure about the name `firstLoader`. Is there something more descriptive we can use here?

Reply:
That's actually either for postcss-loader or sass-loader... I'll call it `postStyleLoader`, what do you think?